packages/thirdweb/src/extensions/erc721/drops/write/claimTo.test.ts

```diff
@@ -0,0 +1,35 @@
+import { type Abi, toFunctionSelector } from "viem";
+import { describe, expect, it } from "vitest";
+import { ANVIL_CHAIN } from "~test/chains.js";
+import { TEST_CONTRACT_URI } from "~test/ipfs-uris.js";
+import { TEST_CLIENT } from "~test/test-clients.js";
+import { TEST_ACCOUNT_B } from "~test/test-wallets.js";
+import { resolveContractAbi } from "../../../../contract/actions/resolve-abi.js";
+import { getContract } from "../../../../contract/contract.js";
+import { deployERC721Contract } from "../../../../extensions/prebuilts/deploy-erc721.js";
+import { isClaimToSupported } from "./claimTo.js";
+
+describe.runIf(process.env.TW_SECRET_KEY)("ERC721: claimTo", () => {
+  it("isClaimToSupported should work", async () => {
+    const contract = getContract({
```

Comment:
Can we run this test against an existing contract? That way we don't rely on deploy which hits IPFS